modules/http/src/main/java/org/mule/module/http/internal/request/DefaultHttpRequester.java

```diff
@@ -360,6 +371,25 @@ private boolean supportsRetry(String httpMethod)
         return retryOnAllMethods || IDEMPOTENT_METHODS.contains(httpMethod);
     }
 
+    /**
+     * Verify if http request entity support retry.
+     * @param request
+     * @return true if entity support retry
+     */
+    private boolean entitySupportRetry(HttpRequest request)
+    {
+        boolean entitySupportRetry = true;
```

Comment:
unify the conditions to make this method clearer. Return true or false according to that.